app/views/users/show.html.haml

```diff
@@ -6,6 +6,11 @@
       %h1.user_profile_header= @user.name
       %p= link_to 'Edit profile', users_edit_profile_path
 
+      - if @user.about_me?
+        %p= @user.about_me
+      -else
+        = 'Your profile is empty. To get the most out of Lend & Tend please take a moment to add to your profile:'
```

Comment:
Interestingly, had no other way of doing this as devise doesn't seperate between facebook signups and logins! 

But yeah should make for some cool design extensibility!